qtconsole/rich_jupyter_widget.py

```diff
@@ -138,7 +138,12 @@ def _handle_execute_result(self, msg):
                 self._append_html(self.output_sep2, True)
             elif 'text/latex' in data:
                 self._pre_image_append(msg, prompt_number)
-                self._append_latex(data['text/latex'], True)
+                try:
+                    self._append_latex(data['text/latex'], True)
+                except Exception as e:
+                    self.log.error("Failed to render latex: '%s'", data['text/latex'], exc_info=True)
+                    self.log.error("Failed to render latex: %s" % e)
```

Comment:
I think the second log call is redundant here - `exc_info=True` in the first one should log the the error that was caught.